File: back/dentalium/src/main/java/com/romoreno/dentalium/infraestructure/adapter/in/controller/rest/assignment/dto/ListAssigmentResponse.java
package com.romoreno.dentalium.infraestructure.adapter.in.controller.rest.assignment.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.ArrayList;
import java.util.Date;
import java.util.List;

@Data
@AllArgsConstructor
public class ListAssigmentResponse {

    private List<Person> assigmentList;

    public ListAssigmentResponse() {
        assigmentList = new ArrayList<>();
    }

    @Data
    @AllArgsConstructor
    @NoArgsConstructor
    public static class Person {
        private String id;
        private String name;
        private Date initDate;
    }

}

File: front/dentalium/src/app/model/api/ListAssigmentResponse.ts
import {Status} from './BudgetResponse';

export class ListAssigmentResponse {
  assigmentList: Person[] = [];

  constructor() {
    this.assigmentList = [];
  }
}

export class Person {

  id: number;
  name: string;
  initDate: Date;

  constructor(id: number, name: string, initDate: Date) {
    this.id = id;
    this.name = name;
    this.initDate = initDate;
  }
}
